libs/platform/src/lib/components/form/multi-input/multi-input.component.ts

```diff
@@ -0,0 +1,150 @@
+import { BACKSPACE, DELETE } from '@angular/cdk/keycodes';
+import {
+    Host,
+    Optional,
+    Self,
+    SkipSelf,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ViewEncapsulation,
+    ViewChild,
+    ViewContainerRef,
+    EventEmitter,
+    forwardRef,
+    Input,
+    Output,
+    TemplateRef
+} from '@angular/core';
+import { NgControl, NgForm } from '@angular/forms';
+
+import { TokenizerComponent, KeyUtil } from '@fundamental-ngx/core';
+
+import { BaseInput } from '../base.input';
+import { FormFieldControl } from '../form-control';
+import { FormField } from '../form-field';
+import { InputGroupComponent } from '../input-group/input-group.component';
+import { InputType } from '../input/input.component';
+
+@Component({
+    selector: 'fdp-multi-input',
+    templateUrl: './multi-input.component.html',
+    styleUrls: ['./multi-input.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [
+        {
+            provide: FormFieldControl,
+            useExisting: forwardRef(() => PlatformMultiInputComponent),
+            multi: true
+        }
+    ]
+})
+export class PlatformMultiInputComponent extends BaseInput {
+    @Input()
+    type: InputType;
+
+    /** Selected values from the list items. */
+    @Input()
+    selected: any[] = [];
+
+    @Input()
+    listTemplate: TemplateRef<any>;
+
+    @ViewChild('listTemplate', { read: ViewContainerRef }) outletRef: ViewContainerRef;
+
+    /** @hidden */
+    @ViewChild(TokenizerComponent)
+    tokenizer;
+
+    /** @hidden */
+    @ViewChild(InputGroupComponent)
+    inputComponent;
+
+    /** @hidden Emits event when the menu is opened/closed */
+    @Output()
+    isOpenChange: EventEmitter<boolean> = new EventEmitter<boolean>();
+
+    public updateSelectedListVariables = [];
+
+    /**
+     * Whether the popover is opened.
+     */
+    isOpen = false;
+
+    private _listVariable: any;
+
+    @Input()
+    set listVariable(selectedValue) {
```

Comment:
keep with other inputs. not after variables.
See angular guidelines.
https://angular.io/guide/styleguide#member-sequence